redhat-operators/codeready-workspaces/codeready-workspaces.1.1.0.csv.yaml

```diff
@@ -0,0 +1,370 @@
+                    image: registry.access.redhat.com/codeready-workspaces/server-operator:1.1
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: "codeready-operator"
+                    command:
+                      - /usr/local/bin/che-operator
+                    ports:
+                      - containerPort: 60000
+                        name: metrics
+                    imagePullPolicy: IfNotPresent
+                restartPolicy: Always
+                terminationGracePeriodSeconds: 5
+                serviceAccountName: codeready-operator
+  customresourcedefinitions:
+    owned:
+      - description: Red Hat CodeReady Workspaces cluster with DB and Auth Server
+        displayName: Red Hat CodeReady Workspaces Cluster
+        kind: CheCluster
+        name: checlusters.org.eclipse.che
+        version: v1
+        specDescriptors:
+          - description: Log in to Red Hat CodeReady Workspaces with OpenShift credentials
+            displayName: OpenShift oAuth
+            path: auth.openShiftoAuth
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+          - description: TLS routes
+            displayName: TLS Mode
+            path: server.tlsSupport
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+        statusDescriptors:
+          - description: Route to access Red Hat CodeReady Workspaces
+            displayName: Red Hat CodeReady Workspaces URL
+            path: cheURL
+            x-descriptors:
+            - urn:alm:descriptor:org.w3:link
+          - description: Route to access Keycloak Admin Console
+            displayName: Red Hat SSO Admin Console URL
+            path: keycloakURL
+            x-descriptors:
+            - urn:alm:descriptor:org.w3:link
+          - description: Red Hat CodeReady Workspaces server version
+            displayName: Red Hat CodeReady Workspaces version
+            path: cheVersion
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:label'
+          - description: The current status of the application
+            displayName: Status
+            path: cheClusterRunning
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes.phase'
+  keywords:
+    - codeready
+    - workspaces
+    - devtools
+    - developer
+    - ide
+    - java
+  displayName: Red Hat CodeReady Workspaces
+  provider:
+    name: Red Hat, Inc.
+    url: https://developers.redhat.com/products/codeready-workspaces/overview/
```

Comment:
Please remove this url from the `provider` field. Links should go into the `spec.links` section

Reply:
Fixed